### PR TITLE
Add works and about me section 

### PR DESCRIPTION
**Pull request by BranBayou**
I created a new branch  Works-and-aboutMyself-section and I added works and about me sections.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -91,6 +95,331 @@ ul li a {
...
+}
+
+.stories-intro p {
+  font-weight: 400;
+  font-size: 16px;
+  color: #091e42;
+  font-family: arial, sans-serif, 'Inter';
+  line-height: 24px;
+  padding: 0 20px;
+}
```
Comment (franclobo) [inline styles.css:145]: - [ ] Please 🙏🏽 align the content of the card to match the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| image | image |

File: styles.css [hunk 2]
```diff
@@ -91,6 +95,331 @@ ul li a {
...
+
+.submit-pro-btn .see-pro {
+  width: 124px;
+  height: 48px;
+  background-color: #ff6b00;
+  gap: 10px;
+  border: none;
+  padding: 0;
+  cursor: pointer;
+}
```
Comment (franclobo) [inline styles.css]: - [ ] Please 🙏🏽 add the correct space between the skills buttons and the see project button and correct the background color of the button it is #fff to be similar to the Figma template 👇🏽 

| Figma template | Your template |
|---|---|
| image | image |

File: styles.css [hunk 3]
```diff
@@ -91,6 +95,331 @@ ul li a {
...
+
+.language ul li,
+.freamworks ul li,
+.skills ul li {
+  padding: 10px 0;
+  font-family: arial, sans-serif, 'Inter';
+  font-weight: 400;
+  font-size: 16px;
 }
```
Comment (franclobo) [inline styles.css]: - [ ] Please 🙏🏽 add a space between the bullet points and the title and a space at the end to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| image | image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @BranBayou, 

# Required Changes ♻️

so-close-jim-halpert

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Good use of tags ✔️ 
- Best practices of HTML & CSS 💯 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
.

File: styles.css [hunk 4]
```diff
@@ -91,6 +95,351 @@ ul li a {
...
+  justify-content: center;
+}
+
+.rect-55 {
+  width: 12px;
+}
+
+.rect-56 {
+  width: 8px;
+}
```
Comment (emmiiorji) [inline styles.css:421]: As requested by the previous reviewer, kindly adjust the space between the bullets and the titles and at the end of the section in order to follow the design.

Your design:
image

Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @BranBayou, 

# Status: Changes Required ♻️

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

BbcanBbcan3GIF

 ## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Proper use of GitHub flow


 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by RaoAkif — APPROVED** [review]
## STATUS: APPROVED 🎉🎉🎉

Hi @BranBayou, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approve-image

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**